src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedIdCheck = (
+    tabs.find(tab => tab.id === selectedTabId) !== undefined
+      ? selectedTabId
+      : tabs[0].id
+  );
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': tab.id === selectedIdCheck })}
```

Comment:
Let's create a variable for `tab.id === selectedTab.id` to avoid duplication of code